ospfd/ospf_auth.c

```diff
@@ -159,6 +159,9 @@ static int ospf_auth_check_md5_digest(struct ospf_interface *oi,
 	struct crypt_key *ck = NULL;
 	uint16_t length = ntohs(ospfh->length);
 
+	if (length < sizeof(struct ospf_header)) /* for coverity's sake */
```

Comment:
Despite this is fine, an error would be useful to print in such a case.

Reply:
Added